src/util/query.js

```diff
@@ -105,7 +103,8 @@ module.exports.build = (allowedFields, {
         boost_mode: 'replace'
       }
     };
-    result.sort.push({ _score: { order: 'desc' } });
+    sortBy.push({ _score: { order: 'desc' } });
```

Comment:
Can _not_ use `'mode': 'max'` here, it throws a parser error for unknown field? I'm not sure I understand why, but it keeps us from using the `actionMap.order` mapping here unfortunately.

Reply:
I hope we can do this better :) Let's talk